.travis.yml

```diff
@@ -5,18 +5,14 @@ cache:
   yarn: true
 git:
   depth: 3
-branches:
-  except:
-  - gh-pages
 stages:
 - name: pr-previews
   if: "(branch = master AND type = pull_request) OR branch != master OR NOT branch =~ /^release-.+$/"
 - name: npm publish
-  if: branch =~ /^release-.+$/ AND type != pull_request
-- name: storybook
-  if: branch =~ /^release-.+$/ AND type != pull_request
+  if: branch = master AND type != pull_request AND tag IS present
```

Comment:
@lychyi Do we want to constrain the content of tags that get released at all?

Reply:
Tagging in general is fine for now. We can reduce the scope by a regex matcher later when we find reasons why we need to tag master other than for releases.